client/components/redux-forms/redux-form-textarea/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { Field } from 'redux-form';
+import { omit } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import FormTextarea from 'components/forms/form-textarea';
+
+class ReduxFormTextarea extends PureComponent {
```

Comment:
There were some [nice improvements](https://github.com/Automattic/wp-calypso/commit/d6a8969d331c66f80f4002f18e8e3a6f0061dbc8) made to `ReduxFormToggle` recently. Could we try to do a similar thing here?